templates/bionic/etc/fail2ban/jail.conf.erb

```diff
@@ -0,0 +1,864 @@
+#
```

Comment:
I'd keep all Ubuntu changes in a separate PR from this CentOS 6 acceptance testing PR.